blocks/core/ff_module/ff_module-task/ff_module-task.xsl

```diff
@@ -14,7 +14,7 @@
             </xsl:if>
         </xsl:attribute>
         <dt class="ff_module-task__item ff_module-task__item--title">
-            <a href="{$data/notice/@href}" class="ff_module-task__link"><xsl:value-of select="$data/notice/htmlMessage"/></a>
+            <a href="{$data/notice/@link_href}" class="ff_module-task__link"><xsl:value-of select="$data/notice/htmlMessage"/></a>
```

Comment:
@simonedeluca-firefly How does this get called — via a variable, or is it fed the XML from the Firefly page directly?